jobs/haproxy/spec

```diff
@@ -330,6 +330,8 @@ properties:
           backend_use_http_health:  true    # optional, defaults to false. enables http based health checks for the backend
           backend_http_health_port: 80      # optional, defaults to the port of the backend server. sets the port for the http backend health check
           backend_http_health_uri:  /health # optional, defaults to /health. sets the URI for backend http health checks
+          backend_health_fall: 3  # optional, defaults to 3 if not set. Number of consecutive unsuccessful health checks required before the server is considered unhealthy from a healthy state.
```

Comment:
It's probably worth mentioning that this is ignore unless backend_use_http_health is true (and same for `backend_health_rise`)

Reply:
Update the description to include the note about the property being ignored when backend_use_http_health is false.